network_cmds fails to build on unstable and master

This bug is very recent, within the last day or two, but affects both unstable and master. Whan I attempt to build `pkgs.darwin.network_cmds`, it first fails with this compilation error:
```
/private/tmp/nix-build-network_cmds-481.20.1.drv-0/network_cmds-481.20.1/unbound/validator/val_secalgo.c:228:8: error: incomplete definition of type 'struct DSA_SIG_st'
        dsasig->r = R;
        ~~~~~~^
```

--- a/pkgs/os-specific/darwin/apple-source-releases/network_cmds/default.nix
+++ b/pkgs/os-specific/darwin/apple-source-releases/network_cmds/default.nix
@@ -1,9 +1,9 @@
 { lib, appleDerivation, xcbuildHook
-, openssl, Librpcsvc, xnu, libpcap, developer_cmds }:
+, libressl, Librpcsvc, xnu, libpcap, developer_cmds }:
 
 appleDerivation {
   nativeBuildInputs = [ xcbuildHook ];
-  buildInputs = [ openssl xnu Librpcsvc libpcap developer_cmds ];
+  buildInputs = [ libressl xnu Librpcsvc libpcap developer_cmds ];
 
   NIX_CFLAGS_COMPILE = " -I./unbound -I${xnu}/Library/Frameworks/System.framework/Headers/";
 

